akd/src/tests/test_errors.rs

```diff
@@ -250,6 +251,16 @@ async fn test_read_during_publish<TC: Configuration>() -> Result<(), AkdError> {
     )
     .unwrap();
 
```

Comment:
Could we also test lookup with `hello3`?

Reply:
Definitely! Will do so in the next revision.